spec/fixtures/odata/nested_groups_metadata.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:DataServices>
+    <Schema Namespace="NEMO" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+      <EntityType Name="Group.*form1*.G1">
+        <Property Name="*q_name2*" Type="Edm.String" Nullable="true"/>
+        <Property Name="*q_name3*" Type="Edm.Int64" Nullable="true"/>
+      </EntityType>
+      <EntityType Name="Group.*form1*.G2">
+        <Property Name="*q_name4*" Type="Edm.String" Nullable="true"/>
+        <Property Name="Group Name (1)" Type="NEMO.Group.*form1*.G2.G1" Nullable="true"/>
```

Comment:
I think I am messing with these names in my PR. Safer to use a `*groupnameN*` style placeholder than to rely on the factory sequences.

Reply:
Will do. I do need a temporary workaround here otherwise `Group Name` and `Group Name` collapse into the same key (so only one gets rendered) but I will add the spec placeholder too.

Reply:
Actually the difference here is that group names are deterministic. They can be hard-coded in the fixture and don't depend on how many other groups have been created up till now. Will add a TODO for when a future commit ensures unique group names that don't collide.